idin/src/main/java/org/frankframework/extensions/idin/IdinSender.java

```diff
@@ -172,11 +237,13 @@ public SenderResult sendMessage(Message message, PipeLineSession session) throws
 			throw new SenderException(e);
 		}
 
+		Communicator communicator = createCommunicator(session);
```

Comment:
The creation of the Communicator needs to be done for every message send?

Reply:
Unfortunately yes, otherwise the returnURL cannot be made dynamic...

Reply:
Ok